lib/Dialect/Torch/Utils/Utils.cpp

```diff
@@ -50,7 +50,7 @@ torch_upstream::ScalarType Torch::getScalarTypeForType(Type type) {
     return torch_upstream::ScalarType::Long;
   if (type.isSignedInteger(32))
     return torch_upstream::ScalarType::Int;
-  if (type.isUnsignedInteger(1))
+  if (type.isUnsignedInteger(1) || type.isSignlessInteger(1))
```

Comment:
Actually, the `type.isUnsignedInteger(1)` can be removed as the dtype can't be `si1`
https://github.com/llvm/torch-mlir/blob/eaf34fa02bd2d90d89e2bee2313c161f417c0709/include/torch-mlir/Dialect/Torch/IR/TorchTypes.td#L155-L167